src/poke_env/environment/abstract_battle.py

```diff
@@ -639,8 +640,22 @@ def _parse_message(self, split_message: List[str]) -> None:
             self._in_team_preview = True
         elif split_message[1] == "gen":
             self._format = split_message[2]
+        elif split_message[1] == "inactive":
+            if "disconnected" in split_message[2]:
+                self._anybody_inactive = True
+            elif "reconnected" in split_message[2]:
+                self._anybody_inactive = False
```

Comment:
I think it could be useful for double battles but I would only use the number of disconnections, not the username as this is used only to predict a message of type player that normally wouldn't be sent. Let me know what you think

Reply:
That makes sense. I think this is good enough for now - if a problem arises because of it, we can take care of it then.